### DISCUSSION TITLE
Can't get readings from obd

### DISCUSSION THREAD

**Discussion by Maddatsun** [agronick/aa-torque#163] [Q&A]
Hi, 
I've just installed AA Torque app on my Android 13 smartphone.
All went well except, when connecting to the car, it won't give me any readings out of the obd.
I have another performance monitor installed and it's working perfectly. 
So, it's not the obd reader...nor torque pro app...not even the smartphone. 
What could it be?

**Comment by agronick** [comment]
Is torque pro showing data? Can you post the logs from AATorque?

**Comment by Maddatsun** [comment]
Hi Kyle,

Torque is working fine. I can show some logs later.
I use torque with another performance monitor app and it's fine..


A terça, 7/01/2025, 02:44, Kyle Agronick ***@***.***>
escreveu:

**Comment by Maddatsun** [comment]
Kyle,

Take a look at those too videos from today.

https://youtube.com/shorts/HnSmj3wlWbI?si=jn8sZDULBUd36BGs

https://youtube.com/shorts/eA80uvHsmj8?si=uzkMvvKGjbPpG6Hw



A terça, 7/01/2025, 02:44, Kyle Agronick ***@***.***>
escreveu:

**Comment by agronick** [comment]
A few things. Your OBD adapter obviously isn't working. You might have a cheap one. Also, that isn't this app. That is performance monitor which was abandoned years ago and this app was based off of. This app is AATorque.

**Comment by Maddatsun** [comment]
Kyle,

My obd reader is working. Yes, it is a ELM327.
Torque pero is working just live other apps with the same reader.
The performance monitor I showed you is not you app. It was just to show
you it is working great.
I can run your app on the smartphone and everything works as far as
settings. Once I open it on Android auto, it opens but doesn't show any
readings.
That's why I asked for help.
I'd rather use your app as it is much nicer and has more options.
I read something about king installer bit I tried and doesn't work. I'm
using Android 13, by the way.
Can you guide me through, please?


A terça, 7/01/2025, 16:33, Kyle Agronick ***@***.***>
escreveu:

**Comment by Maddatsun** [comment]
Hi Kyle,

Yesterday, I tried with a new obd reader from a friend. It doesn't make any
difference. Works with other apps just as mine does. Should be the
installation method, idk!
I installed it directly, without using any other app.
What do you recommend?

Thanks

A terça, 7/01/2025, 16:33, Kyle Agronick ***@***.***>
escreveu:
